sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_client.py

```diff
@@ -103,37 +102,54 @@ def create_translation_job(self, inputs, **kwargs):
             source URL to documents and can contain multiple TranslationTargets (one for each language)
             for the destination to write translated documents.
         :type inputs: List[~azure.ai.translation.document.DocumentTranslationInput]
-        :return: A JobStatusResult with information on the status of the translation job.
-        :rtype: ~azure.ai.translation.document.JobStatusResult
+        :return: An instance of a DocumentTranslationPoller. Call `result()` on the poller
+            object to return a pageable of DocumentStatusResult. A DocumentStatusResult will be
+            returned for each translation on a document.
+        :rtype: DocumentTranslationPoller[ItemPaged[~azure.ai.translation.document.DocumentStatusResult]]
         :raises ~azure.core.exceptions.HttpResponseError:
 
         .. admonition:: Example:
 
-            .. literalinclude:: ../samples/sample_check_document_statuses.py
-                :start-after: [START create_translation_job]
-                :end-before: [END create_translation_job]
+            .. literalinclude:: ../samples/sample_begin_translation.py
+                :start-after: [START begin_translation]
+                :end-before: [END begin_translation]
                 :language: python
                 :dedent: 4
-                :caption: Create a translation job.
+                :caption: Translate the documents in your storage container.
         """
 
-        # submit translation job
-        response_headers = self._client.document_translation._start_translation_initial(  # pylint: disable=protected-access
-            inputs=DocumentTranslationInput._to_generated_list(inputs),  # pylint: disable=protected-access
-            cls=lambda pipeline_response, _, response_headers: response_headers,
-            **kwargs
-        )
-
-        def get_job_id(response_headers):
-            operation_loc_header = response_headers['Operation-Location']
-            return operation_loc_header.split('/')[-1]
+        def deserialization_callback(
+            raw_response, _, headers
+        ):  # pylint: disable=unused-argument
+            translation_status = json.loads(raw_response.http_response.text())
+            return self.list_all_document_statuses(translation_status["id"])
 
-        # get job id from response header
-        job_id = get_job_id(response_headers)
-
-        # get job status
-        return self.get_job_status(job_id)
+        polling_interval = kwargs.pop(
+            "polling_interval", self._client._config.polling_interval
+        )
 
+        continuation_token = kwargs.pop("continuation_token", None)
+        pipeline_response = None
+        if continuation_token:
+            pipeline_response = self._client.document_translation.get_translation_status(
```

Comment:
this is weird I know, but necessary for us to use the job ID as the `continuation_token` instead of pickling the whole response